### DISCUSSION TITLE
Restore filer database from volumedata

### DISCUSSION THREAD

**Discussion by Skysailer** [seaweedfs/seaweedfs#4556] [Q&A]
If I see it correctly, the files stored in the volume can no longer be accessed without the filer database. 
Is there a way to restore the data for the filer database from other metadata, e.g. from the volumes, or is it only possible via a backup of the database?

**Comment by cloudymax** [comment]
For Kubernetes deployments at least, you can backup/restore the filer, master, and volume data by using Restic-based tools like k8up or Velero with existing volume claims 

documented here: https://github.com/seaweedfs/seaweedfs/wiki/Kubernetes-Backups-and-Recovery-with-K8up
